Week-5/Work/index/blackjack.html

```diff
@@ -0,0 +1,245 @@
+          let card1image = response2.cards[0].image;
+          let card2image = response2.cards[1].image;
+          let card1 = response2.cards[0].value;
+          let card2 = response2.cards[1].value;
+          checkNum(card1, cards);
+          checkNum(card2, cards);
+          check(cards)
+          cardsimages.push(card1image, card2image);
+          $("#div").append(`<img src = '${card1image}'/>
+                            <img src = '${card2image}'/>`);
+          $.get(`https://deckofcardsapi.com/api/deck/${response.deck_id}/draw/?count=2`, (response3) => {
+            let card1image = response3.cards[0].image;
+            let card2image = response3.cards[1].image;
+            let card1 = response3.cards[0].value;
+            let card2 = response3.cards[1].value;
+            cards2images.push(card1image, card2image);
+            checkNum(card1, cards2.cards);
+            checkNum(card2, cards2.cards);
+            checkBot(cards2.cards)
+            $("#div2").append(`<img id = 'hidden' src = 'http://ic.pics.livejournal.com/dailyafirmation/691132/529371/529371_original.jpg' height = 300px/>
+                               <img src = '${card2image}' height = 300px/>`);
+          })
+          $("#hit").click(() => {
+            $.get(`https://deckofcardsapi.com/api/deck/${response.deck_id}/draw/?count=1`, (response2) => {
+              let card1image = response2.cards[0].image;
+              let card1 = response2.cards[0].value;
+              checkNum(card1, cards);
+              check(cards);
+              $("#div").append(`<img src = '${card1image}'/>`);
+            })
+          })
+
+          $("#stay").click(() => {
+            checkBot(cards2.cards);
+            if (cards2.total <= 15) {
+              $.get(`https://deckofcardsapi.com/api/deck/${response.deck_id}/draw/?count=1`, (response2) => {
+                let cardnewimage = response2.cards[0].image;
+                let cardnew = response2.cards[0].value;
+                checkNum(cardnew, cards2.cards);
+                checkBot(cards2.cards);
+                $("#div2").append(`<img src = '${cardnewimage}' height = 300px/>`);
+              })
+            } else {
+              if (cards2.total > cardsTotal) {
+                $("#top").append(`<h1 id = 'alert'>Dealer Wins</h1>`)
+                console.log(cards2.total);
+                console.log(cardsTotal);
+                $("#hidden").hide();
+                $("#div2").prepend(`<img src = '${cards2images[0]}' height = 300px/>`)
+
+              } else {
+                $("#top").append(`<h1 id = 'alert'>You Win!</h1>`)
+                console.log(cards2.total);
+                console.log(cardsTotal);
+                $("#hidden").hide();
+                $("#div2").prepend(`<img src = '${cards2images[0]}' height = 300px/>`)
+              }
+
+            }
+          })
+        })
+      });
```

Comment:
Is this a dupe? i didn't read this one b/c it looks like a duplicate. Clean up your files man!